lib/froala-editor-sdk/file.rb

```diff
@@ -18,13 +18,16 @@ class File
     # Default upload path.
     @default_upload_path = "public/uploads/files"
 
+    # Defult file access path
+    @default_file_access_path = "/uploads/"
```

Comment:
I'd like to move this to the `options` param rather than adding it as a completely new parameter. This would help us not to break already implementations.

Reply:
@stefanneculai This is moved to the options param in PR for issue no. 7. Thanks.